nextflow.config

```diff
@@ -24,6 +24,11 @@ params {
     max_multiqc_email_size     = '25.MB'
     multiqc_methods_description = null
 
+    // methylkit options
+    chromhmm                   = 's3://epic-bio-data-us-west-2/ref_genome_index/human/ensembl/release_109/chromhmm/hg38_genome_100_segments.bed'
+    ccre                       = 's3://epic-bio-data-us-west-2/ref_genome_index/human/ensembl/release_109/ccre/GRCh38-cCREs.bed'
+    blacklist                  = 's3://epic-bio-data-us-west-2/ref_genome_index/human/ensembl/release_109/blacklist/hg38-blacklist.v2.bed'
+
```

Comment:
⚠️ Can we change the initialization here to null?
Worried that if someone forgets to specify these at the command line when they kick off a run, we could end up applying stats for human to mouse data